Cubical/HITs/UnorderedProd/Base.agda

```diff
@@ -0,0 +1,41 @@
+{-# OPTIONS --safe #-}
+module Cubical.HITs.UnorderedProd.Base where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+
+private variable
+  ℓ ℓ′ : Level
+  A : Type ℓ
+
+infixr 4 _,_
+
+data UnorderedProd (A : Type ℓ) : Type ℓ where
+  _,_ : (x y : A) → UnorderedProd A
+  swap : (x y : A) → (x , y) ≡ (y , x)
+  trunc : isSet (UnorderedProd A)
+
+module Elim {B : UnorderedProd A → Type ℓ′}
+  (_,*_ : (x y : A) → B (x , y))
+  (swap* : (x y : A) → PathP (λ i → B (swap x y i)) (x ,* y) (y ,* x))
+  (trunc* : (xs : UnorderedProd A) → isSet (B xs)) where
+
+  f : (xs : UnorderedProd A) → B xs
```

Comment:
The module should be anonymous and 'f' should be called 'elim'.
Unfortunately, it is not better in FiniteMultiSets and ListedMultiSets...

Reply:
I have already renamed it.